rules/app.bzl

```diff
@@ -56,12 +57,14 @@ def ios_application(name, apple_library = apple_library, infoplists_by_build_set
     application_kwargs["launch_storyboard"] = application_kwargs.pop("launch_storyboard", library.launch_screen_storyboard_name)
     application_kwargs["families"] = application_kwargs.pop("families", ["iphone", "ipad"])
 
-    import_middleman(name = name + ".import_middleman", deps = library.deps, tags = ["manual"])
+    force_load_direct_deps(name = name + ".__internal__.force_load_direct_deps", deps = library.deps, tags = ["manual"])
```

Comment:
Should we only force load swift_library?

Reply:
Nope we need to force load c++/objc too